runtime/Makefile

```diff
@@ -1,4 +1,6 @@
 PYTHON?=$(shell which python3)
+PYTHON_PREFIX?=$(shell python3 -c "import sys; print(sys.prefix)")
+PYTHON_VERSION?=$(shell python3 -c "import sys; print(f'{sys.version_info.major}.{sys.version_info.minor}')")
```

Comment:
I think it would be safer to fetch the prefix and version by executing the `PYTHON` variable, rather than allowing to customize each of the three variables independently.

Reply:
Fixed in [5b42f56](https://github.com/PennyLaneAI/catalyst/pull/1293/commits/5b42f5611da8f4b5427b374201d7664f02c0f722)

Reply:
I would actually go further and use `:=` instead of `?=` to make it clear it's not intended to be a configurable option :)

Reply:
Good point! Changed in [696a2b9](https://github.com/PennyLaneAI/catalyst/pull/1293/commits/696a2b965b5a37295c49d44d7754007b7dd0b1d2)